app/controllers/contstant_projects_controller.rb

```diff
@@ -0,0 +1,15 @@
+class ContestantProjectsController < ApplicationController
+    def index
+        @contestant_projects = ContestantProject.all
+    end
+
+    def show
+        @contestant_projects = ContestantProject.find(params[:id])
+    end
+
+    private
+
+    def contestant_project_params
+        params.permit(:id, :contestant, :project_id)
+    end
+end
```

Comment:
I'm not sure if I understand the purpose of this controller. We don't have views for this controller, and there were no requirements for views for this controller. We also didn't have a form, so I'm not sure why we need the `contestant_projects_params` method.